pattern-1/README.md

```diff
@@ -0,0 +1,142 @@
+# BOSH release for WSO2 Enterprise Integrator deployment pattern 1
+
+This directory contains the BOSH release implementation and PCF tile creation resources for WSO2 Enterprise Integrator 6.4.0.
+
+![WSO2 Enterprise Integrator 6.4.0 deployment pattern 1](images/pattern-1.png)
+
+The following sections provide general steps required for managing the WSO2 Enterprise Integrator 6.4.0 deployment pattern 1
+BOSH release in a BOSH environment deployed in the desired IaaS.
+
+For step-by-step guidelines to manage the BOSH release and build the PCF tile, refer the following:
+
+## Contents
+
+* [Prerequisites](#prerequisites)
+* [Create the BOSH Release](#create-the-bosh-release)
+* [Build the CF tile](#build-the-cf-tile)
+* [Output](#output)
+* [Delete Deployment](#delete-deployment)
+* [BOSH Release Structure](#bosh-release-structure)
+* [References](#references)
+
+## Prerequisites
+
+1. Install the following software.
+
+    - [BOSH Command Line Interface (CLI) v2+](https://bosh.io/docs/cli-v2.html)
+    - [WSO2 Update Manager (WUM)](http://wso2.com/wum)
```

Comment:
Shall we remove WUM? Since this BOSH release can be used to create deployments with GA packs as well. @msmshariq @Savidude WDYT?